trl/trainer/grpo_trainer.py

```diff
@@ -583,14 +583,14 @@ def data_collator(features):  # No data collation is needed in GRPO
                 f"{self.num_generations}, which is less than the minimum required."
             )
         num_processes = self.accelerator.num_processes
-        effective_batch_size = args.per_device_train_batch_size * num_processes * args.gradient_accumulation_steps
+        effective_batch_size = args.per_device_train_batch_size * num_processes * args.num_mini_batches
```

Comment:
To be precise, I would rename this parameter, which actually corresponds to the generation batch size (`generation_batch_size`?), not the effective batch size.

Reply:
Yes, I agree this is more clear.